src/language-js/options.js

```diff
@@ -108,6 +108,31 @@ module.exports = {
     default: false,
     description: "Always add a line break before else.",
   },
+  breakBeforeStatement: {
```

Comment:
I would like to rename this, as I am writing in the review comment. We should probably update the descriptions as well as discussed above.